File: 3430-count-days-without-meetings/count-days-without-meetings.java
class Solution {
    public int countDays(int days, int[][] meetings) {
        Arrays.sort(meetings,Comparator.comparingInt(a -> a[0]));
        int prevTime=0;
        int ans=0;
        for(int[]meet:meetings){
            int start=meet[0];
            int end=meet[1];
            if(start>prevTime+1){
                ans+=start-prevTime-1;
            }
            prevTime=Math.max(prevTime,end);
        }
        ans+=days-prevTime;
        return ans;
    }
}

File: 3430-count-days-without-meetings/count-days-without-meetings.ts
function countDays(days: number, meetings: number[][]): number {
    let  ans=0;
    let prevTime=0;
    meetings.sort((a, b) => a[0] - b[0]);
    for (const meet of meetings) {
        const start=meet[0];
        const end=meet[1];
        if(start>prevTime+1){
            ans+=start-prevTime-1;
        }
        prevTime=Math.max(end,prevTime);
    }
    ans+=days-prevTime;
    return ans;
};